client/js/screens/UserProfile/components/FaveWays.js

```diff
@@ -0,0 +1,46 @@
+import React, {Component} from 'react';
+import {Text, View, TouchableOpacity, Image} from 'react-native';
+import styles from '../styles';
+import source from './source';
+
+class FaveWays extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      style: styles.icon,
+      image: true,
+    };
+  }
+  componentWillMount() {
```

Comment:
componentWillMount lifecycle method is deprecated. Please use componentDidMount instead.

Reply:
Updated